src/Form.php

```diff
@@ -159,8 +166,35 @@ public function __construct($defaults = [])
      */
     public function init(): void
     {
+        // DEBUG FOR NESTED FORMS
+        // see https://github.com/atk4/ui/pull/1219
+        if (isset($this->app)) {
+            $this->template = $this->app->loadTemplate($this->defaultTemplate, new class() extends Template {
+                /** @var Form */
+                public $form;
+
+                protected function recursiveRender($template)
+                {
+                    $out = parent::recursiveRender($template);
+
+                    return $this->fixInnerHtml($out);
+                }
+
+                private function fixInnerHtml(string $str): string
+                {
+                    $innerFormTags = ['button', 'datalist', 'fieldset', 'input', 'keygen', 'label', 'legend', 'meter', 'optgroup', 'option', 'output', 'progress', 'select', 'textarea'];
+                    $str = preg_replace('~<(' . implode('|', $innerFormTags) . ')(?!\w| form=")~i', '$0 form="' . $this->form->formElement->name . '"', $str);
```

Comment:
Is replace like this ok for production? It does not replace anything if `form` attribute is presented, but it handles every case, even if no `Form\Field` element was used.

Nested forms should be supported as well.